x-pack/plugins/alerting/common/index.ts

```diff
@@ -25,14 +25,6 @@ export type {
   RuleTaskState,
   RuleTaskParams,
 } from '@kbn/alerting-state-types';
-export {
-  rawAlertInstance,
-  DateFromString,
-  wrappedStateRt,
-  ActionsCompletion,
-  ruleStateSchema,
-  ruleParamsSchema,
-} from '@kbn/alerting-state-types';
```

Comment:
The modifications I was doing to `@kbn/alerting-state-types` caused the page bundle size of the `alerting` plugin to exceed the threshold. In fact this code is only needed on the server side so I modified all the places requiring these objects to get them directly from the package.